mono/metadata/mono-perfcounters.c

```diff
@@ -1304,52 +1307,52 @@ custom_get_impl (SharedCategory *cat, MonoString *counter, MonoString* instance,
 }
 
 static const CategoryDesc*
-find_category (MonoString *category)
+find_category (const gunichar2* category, int category_length)
 {
 	int i;
+	const CategoryDesc* desc = predef_categories;
 	for (i = 0; i < NUM_CATEGORIES; ++i) {
-		if (mono_string_compare_ascii (category, predef_categories [i].name) == 0)
-			return &predef_categories [i];
+		if (perfcounter_utf16_equal_ascii (category, category_length, desc->name, desc->name_length))
+			return desc;
+		++desc;
 	}
 	return NULL;
 }
 
 void*
-mono_perfcounter_get_impl (MonoString* category, MonoString* counter, MonoString* instance,
-		MonoString* machine, int *type, MonoBoolean *custom)
+ves_icall_System_Diagnostics_PerformanceCounter_GetImpl (
```

Comment:
Please use `gint32` instead of `int` for parameters received from managed, this is to guarantee the size of the argument. Also, coding convention wants that we do not break line right after the parenthesis, and that we break line only when we reach between the column 100 or 120.

Reply:
Ah, int32 I forgot. My editor has a guide at column 80, I'll move it out further.